etc/newsboat.profile

```diff
@@ -0,0 +1,46 @@
+# Firejail profile for Newsboat
+# Description: RSS program
+# This file is overwritten after every install/update
+# Persistent local customizations
+include newsboat.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.newsboat
+whitelist ${HOME}/.newsboat
```

Comment:
whitelist need mkdir and kann be moved down.

You can also add the other things from [here](https://gist.github.com/rusty-snake/8652584e91c4ef1de3e7b531e45de2e2).